src/app/tests/suites/TestUserLabelClusterConstraints.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: User Label Cluster Tests
+
+config:
+    nodeId: 0x12344321
+    cluster: "User Label"
+    endpoint: 0
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Attempt to write overly long item for label"
+      command: "writeAttribute"
+      attribute: "label list"
+      arguments:
+          value:
+              [
+                  {
+                      label: "this is longer than sixteen characters",
+                      value: "bedroom 2",
+                  },
+              ]
+      response:
+          error: FAILURE
```

Comment:
Failure is the wrong IM response. It has to map to `CONSTRAINT_ERROR` as the error, since this is the case of writing an out of bound value to an *attribute*. The code in the user-label-server does not properly handle converting to IM error and seems to cause FAILURE if this passes at all

Reply:
I will check for changing that in a separate PR on master. For now this patches SVE branch, so should probably be just allowed through.